src/title/TitleGroup.js

```diff
@@ -22,7 +22,7 @@ const componentStyles = createStyleSheet({
 export default function TitleGroup(props: Props) {
   const { userIds } = props;
   return (
-    <View style={styles.navWrapper}>
+    <ScrollView horizontal showsHorizontalScrollIndicator={false}>
```

Comment:
What's the reason for removing the `style={styles.navWrapper}`? 🙂 I don't think I'm attached to it (though I haven't thought super hard about it yet). But if we remove it, I'd like to record a reason for doing so.

…Ah, I see that I get the following error if I try to add that back in on the `ScrollView`:

> Invariant Violation: ScrollView child layout (["alignItems","justifyContent"]) must be applied through the contentContainerStyle prop.

What happens if you follow that recommendation, and apply `alignItems` and `justifyContent` through the `contentContinerStyle` prop, instead of the `style` prop?

Or is there some other motivation for dropping `styles.navWrapper`?

Reply:
>Or is there some other motivation for dropping styles.navWrapper?

I thought it was of no use as the ScrollView horizontal is equivalent to `flexDirection: row`, and the spacing is taken care of by the `marginRight` property of the avatars

Reply:
> ScrollView horizontal is equivalent to `flexDirection: row`

This may sound a bit nit-picky, but this isn't really true. 🙂 Strictly speaking, they aren't equivalent: passing `true` for `horizontal` will help us have horizontal scrolling behavior, while `flexDirection: 'row'` will not (and it hasn't been; we've been setting that, and we don't have horizontal scrolling). Since we know of at least that difference, it might be useful to explore more, to help avoid accidentally introducing bugs.

> What happens if you follow that recommendation, and apply `alignItems` and `justifyContent` through the `contentContainerStyle` prop, instead of the `style` prop?

Have you tried this out? 🙂 Do you notice anything?